napari/_qt/_qapp_model/_tests/test_debug_menu.py

```diff
@@ -0,0 +1,131 @@
+import importlib
+from unittest import mock
+
+import pytest
+
+from napari._app_model import get_app
+from napari._qt._qapp_model._tests.utils import get_submenu_action
+
+
+@pytest.fixture(params=[True, False])
+def perfmon_activation(monkeypatch, request):
+    if request.param:
+        # If perfmon needs to be active add env var and reload modules
+        monkeypatch.setenv('NAPARI_PERFMON', '1')
+        from napari.utils import perf
+
+        importlib.reload(perf._timers)
+        importlib.reload(perf)
+
+        # Check `USE_PERFMON` values correspond to env var state
+        assert perf._timers.USE_PERFMON
+        assert perf.USE_PERFMON
```

Comment:
@DragaDoncila any idea why we use `perf.USE_PERFMON` almost everywhere except in `perf._timers.py` ?

Reply:
I have none at all. I'm afraid the first I saw this code was when you mentioned the `Debug` menu so... 😅 @jni wanted to improve this menu though, and add more functionality, so we can maybe do a deeper dive once we look at that.